### ISSUE TITLE
Tallies in `dev` fail queue

### ISSUE THREAD

**Issue by hannes-ucsc** [DataBiosphere/azul#1845]
Found these tallies in the fail queue. They need to be investigated.

```
{
    "queue": "https://queue.amazonaws.com/122796619775/azul-fail-dev.fifo",
    "messages": [
        {
            "MessageId": "704531f0-02f9-42b4-b121-9410f5dc9312",
            "ReceiptHandle": "AQEBdBWFPLSjGWmfYrr4hYy0W0GJoVwv7qgtEBaRfrF6XPHq1P+HZCYFS+xgedrXqZxr1CvoljVMJ1Yr7PtYI3Mg8HKFeTCPu1FZN10d7i868zxzBUXY5V9tNYo5mW2Kxvy9Y7O8dC1amiwgvqmGu62Pixavb0taJOJ0psQuAaaGv/V1QJgbqaRJHYHXCQ4BQA1u/tMzeo32P0i+m09WXZDhl0j9J8xjfSdFd8wHNsFlZu8KLQsSPPZXspxvnIEnp1KS5kCUULYUUgHSxpC9wXULDO08wAgTmXESvjU7Eqc2dEo=",
            "MD5OfBody": "1730d36ab23e53ca5307da939b0b2121",
            "Body": {
                "entity_type": "samples",
                "entity_id": "1e8ea7a3-8545-4ca5-82e3-1132ce7c940a",
                "num_contributions": 94
            },
            "Attributes": {
                "SenderId": "AROARZFZ7W77VNWF46R45:azul-indexer-dev-write",
                "ApproximateFirstReceiveTimestamp": "1589502785345",
                "ApproximateReceiveCount": "14",
                "SentTimestamp": "1589502775338",
                "SequenceNumber": "18853657580439535616",
                "MessageDeduplicationId": "704531f0-02f9-42b4-b121-9410f5dc9312",
                "MessageGroupId": "1e8ea7a3-8545-4ca5-82e3-1132ce7c940a"
            },
            "_Attributes": {
                "SentTimestamp": "2020-05-14T17:32:55.338000-07:00",
                "ApproximateFirstReceiveTimestamp": "2020-05-14T17:33:05.345000-07:00"
            }
        },
        {
            "MessageId": "9fe2956b-0198-4b92-b79d-15611a6bf4b6",
            "ReceiptHandle": "AQEBINEKTcThPhSgyh0yKpg0ZabIGH9nA/XIj0BxbN+pWjm8hMMLjxmBYu5PKfd6DMW3pmssdAgEdTfDzIo7iBCZNAaM0B03+lSxhe1f/sCH+SVMPzOmxLFx9Ui8lis3U4i6xXA5LlSK7zt3Q7ECgvP4CWs4FyRPZWAqTSmMKQAoL1kzepShKftXBFy3nfinqwUcsU+XmVr/GLkvIZadC/V/CwbLpw5cHHk+kc7ipjNc6drHFvTKSd0CnkmN0lM29LVAC+A0BwcbZxxaxLf91fVMzW4xJRzqpg8NZTsAvu2Ljo0=",
            "MD5OfBody": "ad8c7feb18283d4b60ede19597d3148a",
            "Body": {
                "entity_type": "samples",
                "entity_id": "8ee0baf8-becf-48c7-93ba-8d5603567a79",
                "num_contributions": 7
            },
            "Attributes": {
                "SenderId": "AROARZFZ7W77VNWF46R45:azul-indexer-dev-write",
                "ApproximateFirstReceiveTimestamp": "1589502785345",
                "ApproximateReceiveCount": "14",
                "SentTimestamp": "1589502775338",
                "SequenceNumber": "18853657580431087872",
                "MessageDeduplicationId": "9fe2956b-0198-4b92-b79d-15611a6bf4b6",
                "MessageGroupId": "8ee0baf8-becf-48c7-93ba-8d5603567a79"
            },
            "_Attributes": {
                "SentTimestamp": "2020-05-14T17:32:55.338000-07:00",
                "ApproximateFirstReceiveTimestamp": "2020-05-14T17:33:05.345000-07:00"
            }
        }
    ]
}
```

**Comment by amarjandu** [comment]
Found some errors with EventualConsistancyExceptions for `"entity_id": "8ee0baf8-becf-48c7-93ba-8d5603567a79"`:

```
[ERROR] EventualConsistencyException: ('Could not find all expected contributions.', {EntityReference(entity_type='cell_suspensions', entity_id='f0fa5510-a68c-491e-8294-8638b66a1fce'): 1, EntityReference(entity_type='files', entity_id='e1151ca9-95a1-41ed-9752-847588162241'): 1, EntityReference(entity_type='files', entity_id='8c97a0b0-30fb-4244-afb0-9eb5e3a009dc'): 1, EntityReference(entity_type='samples', entity_id='8ee0baf8-becf-48c7-93ba-8d5603567a79'): 3, EntityReference(entity_type='files', entity_id='14a2c083-dad8-4747-91c6-9630ff960edf'): 1, EntityReference(entity_type='bundles', entity_id='14b39590-4cac-49ba-8040-3bc553fc9b83'): 1, EntityReference(entity_type='files', entity_id='792b7d5a-0872-4cee-9e33-606bb38bc985'): 1, EntityReference(entity_type='cell_suspensions', entity_id='53ad70fb-f11d-42e9-aa2b-d9dbff45968d'): 1, EntityReference(entity_type='files', entity_id='c875eef5-2dde-46f6-bae3-63d448b2f267'): 1, EntityReference(entity_type='bundles', entity_id='0287c2ce-2740-4803-9274-bd255f8ad8f0'): 1}, Counter({EntityReference(entity_type='samples', entity_id='8ee0baf8-becf-48c7-93ba-8d5603567a79'): 6, EntityReference(entity_type='cell_suspensions', entity_id='53ad70fb-f11d-42e9-aa2b-d9dbff45968d'): 2, EntityReference(entity_type='cell_suspensions', entity_id='f0fa5510-a68c-491e-8294-8638b66a1fce'): 1, EntityReference(entity_type='files', entity_id='14a2c083-dad8-4747-91c6-9630ff960edf'): 1, EntityReference(entity_type='files', entity_id='8c97a0b0-30fb-4244-afb0-9eb5e3a009dc'): 1, EntityReference(entity_type='files', entity_id='c875eef5-2dde-46f6-bae3-63d448b2f267'): 1, EntityReference(entity_type='files', entity_id='e1151ca9-95a1-41ed-9752-847588162241'): 1, EntityReference(entity_type='files', entity_id='792b7d5a-0872-4cee-9e33-606bb38bc985'): 1}))
Traceback (most recent call last):
  File "/var/task/app.py", line 102, in new_handler
    return old_handler(self, event, context)
  File "/var/task/chalice/app.py", line 1430, in __call__
    return self.func(event_obj)
  File "/var/task/app.py", line 116, in write
    app.index_controller.aggregate(event)
  File "/var/task/azul/indexer/index_controller.py", line 226, in aggregate
    self.index_service.aggregate(tallies)
  File "/var/task/azul/indexer/index_service.py", line 192, in aggregate
    raise EventualConsistencyException(message, *args)
``` 

Note the timestamps for these errors dont match with the timestamps for the message in SQS.

**Comment by amarjandu** [comment]
Appear to be transport errors present within the `azul_aggregate_dev` document.

```
[WARNING]	2020-05-29T04:48:23.341Z	ddc77077-e0a5-580b-a503-273d76f170fe	There was a general error with document DocumentCoordinates(document_index='azul_files_aggregate_dev', document_id='99462f12-2491-4e02-9d0f-4636a2994598'): TransportError(503, 'circuit_breaking_exception', {'error': {'root_cause': [{'type': 'circuit_breaking_exception', 'reason': '[parent] Data too large, data for [<transport_request>] would be [16301780817/15.1gb], which is larger than the limit of [16287753830/15.1gb], real usage: [16301777400/15.1gb], new bytes reserved: [3417/3.3kb]', 'bytes_wanted': 16301780817, 'bytes_limit': 16287753830}], 'type': 'circuit_breaking_exception', 'reason': '[parent] Data too large, data for [<transport_request>] would be [16301780817/15.1gb], which is larger than the limit of [16287753830/15.1gb], real usage: [16301777400/15.1gb], new bytes reserved: [3417/3.3kb]', 'bytes_wanted': 16301780817, 'bytes_limit': 16287753830}, 'status': 503}). Total # of errors: 1, giving up.
```

limit of 15.1gb, requested 16.3gb,
when looking at the jvm-memory-pressure chart for elasticsearch, there is a spike for max usage at ~95% during the reindex operation, the chart does not indicate time that can be related to the error timestamps.

**Comment by amarjandu** [comment]
**CloudWatch Logs Insights**  
region: us-east-1  
log-group-names: /aws/lambda/azul-indexer-dev-aggregate, /aws/lambda/azul-indexer-dev, /aws/lambda/azul-indexer-dev-aggregate_retry, /aws/lambda/azul-indexer-dev-contribute, /aws/lambda/azul-indexer-dev-index, /aws/lambda/azul-indexer-dev-indexercachehealth, /aws/lambda/azul-indexer-dev-nudge, /aws/lambda/azul-indexer-dev-write  
start-time: 2020-05-14T07:00:00.000Z  
end-time: 2020-05-16T06:59:59.000Z  
query-string:
```
fields @timestamp, @message
|filter @message like '6bdbd6a3-0324-5e8a-acb6-8194f372bf76'
| sort @timestamp desc

```
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
|       @timestamp        |                                                                                                                     @message                                                                                                                      |
|-------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| 2020-05-15 01:19:35.743 | END RequestId: 6bdbd6a3-0324-5e8a-acb6-8194f372bf76                                                                                                                                                                                               |
| 2020-05-15 01:19:35.743 | REPORT RequestId: 6bdbd6a3-0324-5e8a-acb6-8194f372bf76 Duration: 203.52 ms Billed Duration: 300 ms Memory Size: 3008 MB Max Memory Used: 363 MB                                                                                                   |
| 2020-05-15 01:19:35.739 | [DEBUG] 2020-05-15T01:19:35.739Z 6bdbd6a3-0324-5e8a-acb6-8194f372bf76 Number of contributions read, by entity: {'samples/1e8ea7a3-8545-4ca5-82e3-1132ce7c940a': 91, 'samples/8ee0baf8-becf-48c7-93ba-8d5603567a79': 95}                           |
| 2020-05-15 01:19:35.738 | [INFO] 2020-05-15T01:19:35.738Z 6bdbd6a3-0324-5e8a-acb6-8194f372bf76 Read 186 contribution(s).                                                                                                                                                    |
| 2020-05-15 01:19:35.618 | [INFO] 2020-05-15T01:19:35.618Z 6bdbd6a3-0324-5e8a-acb6-8194f372bf76 GET https://search-azul-index-dev-atlcgwfjbk6dvrtmiaskbs2swa.us-east-1.es.amazonaws.com:443/azul_samples_dev/doc/_search?size=1000 [status:200 request:0.052s]               |
| 2020-05-15 01:19:35.565 | [INFO] 2020-05-15T01:19:35.565Z 6bdbd6a3-0324-5e8a-acb6-8194f372bf76 Reading 190 expected contribution(s) using search().                                                                                                                         |
| 2020-05-15 01:19:35.564 | [INFO] 2020-05-15T01:19:35.564Z 6bdbd6a3-0324-5e8a-acb6-8194f372bf76 GET https://search-azul-index-dev-atlcgwfjbk6dvrtmiaskbs2swa.us-east-1.es.amazonaws.com:443/_mget?_source_include=entity_id%2Cnum_contributions [status:200 request:0.024s]  |
| 2020-05-15 01:19:35.540 | [INFO] 2020-05-15T01:19:35.540Z 6bdbd6a3-0324-5e8a-acb6-8194f372bf76 Attempt 10 of handling 94 contribution(s) for entity samples/1e8ea7a3-8545-4ca5-82e3-1132ce7c940a                                                                            |
| 2020-05-15 01:19:35.540 | [INFO] 2020-05-15T01:19:35.540Z 6bdbd6a3-0324-5e8a-acb6-8194f372bf76 Attempt 10 of handling 7 contribution(s) for entity samples/8ee0baf8-becf-48c7-93ba-8d5603567a79                                                                             |
| 2020-05-15 01:19:35.540 | [INFO] 2020-05-15T01:19:35.540Z 6bdbd6a3-0324-5e8a-acb6-8194f372bf76 Aggregating 94 contribution(s) to entity samples/1e8ea7a3-8545-4ca5-82e3-1132ce7c940a                                                                                        |
| 2020-05-15 01:19:35.540 | [INFO] 2020-05-15T01:19:35.540Z 6bdbd6a3-0324-5e8a-acb6-8194f372bf76 Aggregating 7 contribution(s) to entity samples/8ee0baf8-becf-48c7-93ba-8d5603567a79                                                                                         |
| 2020-05-15 01:19:35.537 | START RequestId: 6bdbd6a3-0324-5e8a-acb6-8194f372bf76 Version: $LATEST                                                                                                                                                                            |
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

lambda-invocation-id: `2396078a-094e-5b5b-91ec-50bde66a6998` also has the same difference in the number of contributions expected and what was actually provided back to the indexer. Missing 4 contributions.

**Comment by amarjandu** [comment]
**CloudWatch Logs Insights**  
region: us-east-1  
log-group-names: /aws/lambda/azul-indexer-dev-aggregate, /aws/lambda/azul-indexer-dev, /aws/lambda/azul-indexer-dev-aggregate_retry, /aws/lambda/azul-indexer-dev-contribute, /aws/lambda/azul-indexer-dev-index, /aws/lambda/azul-indexer-dev-indexercachehealth, /aws/lambda/azul-indexer-dev-nudge, /aws/lambda/azul-indexer-dev-write  
start-time: 2020-05-15T00:57:05.235Z  
end-time: 2020-05-15T01:03:27.131Z  
query-string:
```
fields @timestamp, @message

| sort @timestamp desc

```
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
|       @timestamp        |                                                                                                                                                                                                                                                                                                                 @message                                                                                                                                                                                                                                                                                                                  |
|-------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| 2020-05-15 01:02:49.207 | END RequestId: d0aaa692-da9d-4281-ba45-0058b8b75d9d                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
| 2020-05-15 01:02:49.207 | REPORT RequestId: d0aaa692-da9d-4281-ba45-0058b8b75d9d Duration: 925.86 ms Billed Duration: 1000 ms Memory Size: 128 MB Max Memory Used: 96 MB                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
| 2020-05-15 01:02:48.926 | [INFO] 2020-05-15T01:02:48.926Z d0aaa692-da9d-4281-ba45-0058b8b75d9d HEAD https://search-azul-index-dev-atlcgwfjbk6dvrtmiaskbs2swa.us-east-1.es.amazonaws.com:443/ [status:200 request:0.058s]                                                                                                                                                                                                                                                                                                                                                                                                                                            |
| 2020-05-15 01:02:48.279 | START RequestId: d0aaa692-da9d-4281-ba45-0058b8b75d9d Version: $LATEST                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
| 2020-05-15 01:01:48.787 | END RequestId: b3cd4ea3-9d24-4448-807a-dfcf32f3248b                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
| 2020-05-15 01:01:48.787 | REPORT RequestId: b3cd4ea3-9d24-4448-807a-dfcf32f3248b Duration: 894.52 ms Billed Duration: 900 ms Memory Size: 128 MB Max Memory Used: 96 MB                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
| 2020-05-15 01:01:48.506 | [INFO] 2020-05-15T01:01:48.506Z b3cd4ea3-9d24-4448-807a-dfcf32f3248b HEAD https://search-azul-index-dev-atlcgwfjbk6dvrtmiaskbs2swa.us-east-1.es.amazonaws.com:443/ [status:200 request:0.078s]                                                                                                                                                                                                                                                                                                                                                                                                                                            |
| 2020-05-15 01:01:47.890 | START RequestId: b3cd4ea3-9d24-4448-807a-dfcf32f3248b Version: $LATEST                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
| 2020-05-15 01:00:49.338 | END RequestId: cc849ff6-cfbe-4b1e-b96d-7db96be3aaf8                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
| 2020-05-15 01:00:49.338 | REPORT RequestId: cc849ff6-cfbe-4b1e-b96d-7db96be3aaf8 Duration: 952.04 ms Billed Duration: 1000 ms Memory Size: 128 MB Max Memory Used: 96 MB                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
| 2020-05-15 01:00:49.046 | [INFO] 2020-05-15T01:00:49.45Z cc849ff6-cfbe-4b1e-b96d-7db96be3aaf8 HEAD https://search-azul-index-dev-atlcgwfjbk6dvrtmiaskbs2swa.us-east-1.es.amazonaws.com:443/ [status:200 request:0.078s]                                                                                                                                                                                                                                                                                                                                                                                                                                             |
| 2020-05-15 01:00:48.382 | START RequestId: cc849ff6-cfbe-4b1e-b96d-7db96be3aaf8 Version: $LATEST                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
| 2020-05-15 00:59:48.831 | END RequestId: ad4c9665-b1c6-4df1-b5cd-c02c46c23b57                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
| 2020-05-15 00:59:48.831 | REPORT RequestId: ad4c9665-b1c6-4df1-b5cd-c02c46c23b57 Duration: 890.01 ms Billed Duration: 900 ms Memory Size: 128 MB Max Memory Used: 96 MB                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
| 2020-05-15 00:59:48.551 | [INFO] 2020-05-15T00:59:48.550Z ad4c9665-b1c6-4df1-b5cd-c02c46c23b57 HEAD https://search-azul-index-dev-atlcgwfjbk6dvrtmiaskbs2swa.us-east-1.es.amazonaws.com:443/ [status:200 request:0.006s]                                                                                                                                                                                                                                                                                                                                                                                                                                            |
| 2020-05-15 00:59:47.939 | START RequestId: ad4c9665-b1c6-4df1-b5cd-c02c46c23b57 Version: $LATEST                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
| 2020-05-15 00:58:58.966 | END RequestId: 0281814b-c2cb-5e23-8b3d-d67c29b5a422                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
| 2020-05-15 00:58:58.966 | REPORT RequestId: 0281814b-c2cb-5e23-8b3d-d67c29b5a422 Duration: 227.43 ms Billed Duration: 300 ms Memory Size: 3008 MB Max Memory Used: 363 MB                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
| 2020-05-15 00:58:58.963 | : AssertionError Traceback (most recent call last):   File "/var/task/app.py", line 102, in new_handler     return old_handler(self, event, context)   File "/var/task/chalice/app.py", line 1422, in __call__     return self.func(event_obj)   File "/var/task/app.py", line 116, in write     app.index_controller.aggregate(event)   File "/var/task/azul/indexer/index_controller.py", line 166, in aggregate     indexer.aggregate(tallies)   File "/var/task/azul/indexer/__init__.py", line 250, in aggregate     assert all(tallies[entity] <= actual_tally for entity, actual_tally in actual_tallies.items()) AssertionError   |
| 2020-05-15 00:58:58.962 | [DEBUG] 2020-05-15T00:58:58.961Z 0281814b-c2cb-5e23-8b3d-d67c29b5a422 Number of contributions read, by entity: {'samples/1e8ea7a3-8545-4ca5-82e3-1132ce7c940a': 91, 'samples/8ee0baf8-becf-48c7-93ba-8d5603567a79': 95}                                                                                                                                                                                                                                                                                                                                                                                                                   |
| 2020-05-15 00:58:58.961 | [INFO] 2020-05-15T00:58:58.961Z 0281814b-c2cb-5e23-8b3d-d67c29b5a422 Read 186 contribution(s).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
| 2020-05-15 00:58:58.840 | [INFO] 2020-05-15T00:58:58.840Z 0281814b-c2cb-5e23-8b3d-d67c29b5a422 GET https://search-azul-index-dev-atlcgwfjbk6dvrtmiaskbs2swa.us-east-1.es.amazonaws.com:443/azul_samples_dev/doc/_search?size=1000 [status:200 request:0.051s]                                                                                                                                                                                                                                                                                                                                                                                                       |
| 2020-05-15 00:58:58.788 | [INFO] 2020-05-15T00:58:58.787Z 0281814b-c2cb-5e23-8b3d-d67c29b5a422 GET https://search-azul-index-dev-atlcgwfjbk6dvrtmiaskbs2swa.us-east-1.es.amazonaws.com:443/_mget?_source_include=entity_id%2Cnum_contributions [status:200 request:0.048s]                                                                                                                                                                                                                                                                                                                                                                                          |
| 2020-05-15 00:58:58.788 | [INFO] 2020-05-15T00:58:58.788Z 0281814b-c2cb-5e23-8b3d-d67c29b5a422 Reading 190 expected contribution(s) using search().                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
| 2020-05-15 00:58:58.739 | [INFO] 2020-05-15T00:58:58.738Z 0281814b-c2cb-5e23-8b3d-d67c29b5a422 Attempt 6 of handling 94 contribution(s) for entity samples/1e8ea7a3-8545-4ca5-82e3-1132ce7c940a                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
| 2020-05-15 00:58:58.739 | [INFO] 2020-05-15T00:58:58.739Z 0281814b-c2cb-5e23-8b3d-d67c29b5a422 Attempt 6 of handling 7 contribution(s) for entity samples/8ee0baf8-becf-48c7-93ba-8d5603567a79                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
| 2020-05-15 00:58:58.739 | [INFO] 2020-05-15T00:58:58.739Z 0281814b-c2cb-5e23-8b3d-d67c29b5a422 Aggregating 94 contribution(s) to entity samples/1e8ea7a3-8545-4ca5-82e3-1132ce7c940a                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
| 2020-05-15 00:58:58.739 | [INFO] 2020-05-15T00:58:58.739Z 0281814b-c2cb-5e23-8b3d-d67c29b5a422 Aggregating 7 contribution(s) to entity samples/8ee0baf8-becf-48c7-93ba-8d5603567a79                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
| 2020-05-15 00:58:58.736 | START RequestId: 0281814b-c2cb-5e23-8b3d-d67c29b5a422 Version: $LATEST                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
| 2020-05-15 00:58:49.709 | END RequestId: a88e754b-abae-454a-a8de-7691fe5f6ce3                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
| 2020-05-15 00:58:49.709 | REPORT RequestId: a88e754b-abae-454a-a8de-7691fe5f6ce3 Duration: 1463.06 ms Billed Duration: 1500 ms Memory Size: 128 MB Max Memory Used: 96 MB                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
| 2020-05-15 00:58:49.385 | [INFO] 2020-05-15T00:58:49.384Z a88e754b-abae-454a-a8de-7691fe5f6ce3 HEAD https://search-azul-index-dev-atlcgwfjbk6dvrtmiaskbs2swa.us-east-1.es.amazonaws.com:443/ [status:200 request:0.058s]                                                                                                                                                                                                                                                                                                                                                                                                                                            |
| 2020-05-15 00:58:48.243 | START RequestId: a88e754b-abae-454a-a8de-7691fe5f6ce3 Version: $LATEST                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
| 2020-05-15 00:57:49.707 | END RequestId: e8c5870a-a840-493b-9b36-de9b4be8c002                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
| 2020-05-15 00:57:49.707 | REPORT RequestId: e8c5870a-a840-493b-9b36-de9b4be8c002 Duration: 1530.27 ms Billed Duration: 1600 ms Memory Size: 128 MB Max Memory Used: 96 MB                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
| 2020-05-15 00:57:48.804 | [INFO] 2020-05-15T00:57:48.804Z e8c5870a-a840-493b-9b36-de9b4be8c002 HEAD https://search-azul-index-dev-atlcgwfjbk6dvrtmiaskbs2swa.us-east-1.es.amazonaws.com:443/ [status:200 request:0.078s]                                                                                                                                                                                                                                                                                                                                                                                                                                            |
| 2020-05-15 00:57:48.171 | START RequestId: e8c5870a-a840-493b-9b36-de9b4be8c002 Version: $LATEST                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

Logs for complete lambda invocation + a little time buffer around the execution.

**Comment by hannes-ucsc** [comment]
We need to think long and hard about as to whether that assertion should instead be a conditional `raise EventualConsistencyException` and how we are going to prevent it from probabilistically exceeding the ten retries.

**Comment by hannes-ucsc** [comment]
In https://github.com/DataBiosphere/azul/commit/985fc0e51842df6deb5bf27b0bcecc64b997d877#diff-35457eca5a95377ed88a340ff27e9ed8020c1c456fba040d423afe8fc536ee0fR252 for #2330 I attempted to change this from an `assert` to raising `EventualConsistencyException` but that change had to be [reverted](https://github.com/DataBiosphere/azul/issues/2330#issuecomment-718900470). This issue will be addressed in some form by a working solution for #2330. Closing this one.
